html/webSocket.js

```diff
@@ -0,0 +1,46 @@
+/* global submitModalNewCommand */
+
+let webSocket;
+
+$(document).ready(function () {
+    const webSocketStatusElement = document.getElementById("webSocketStatus");
+    webSocket = new WebSocket("ws://" + window.location.host + "/ws");
+
+    // Web socket open
+    webSocket.onopen = function () {
+        webSocketStatusElement.innerHTML = "Web socket status: <b>Connected</b>";
+    };
+
+    // Web socket receive message
+    webSocket.onmessage = function (e) {
+        let data = JSON.parse(e.data);
+
+        if (data.kind === messageKindCommand) {
+            if (data.operation === messageOperationWrite) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['messageOperationWrite' is not defined.](https://app.codacy.com/manual/ttimt/Short_Cutteer/pullRequest?prid=4845233)